common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -239,6 +242,20 @@ def multidependency_intro
         "dependencies needed to be updated together."
       end
 
+      def vulnerable_multidependency_intro
+        msg = "Bumps #{dependency_links[0]} to a non-vulnerable version "
```

Comment:
There's a few other cases here:
1. The vulnerable dependency is removed rather than bumped.
2. Multi dependency PRs that are updating multiple direct dependencies

Not necessarily a blocker but it be useful to have some examples of what these PRs would look like.